timemachine/fe/single_topology.py

```diff
@@ -916,8 +916,10 @@ def combine_confs(self, x_a, x_b):
             Combined conformation
 
         """
-        warnings.warn("combine confs is deprecated for SingleTopology", DeprecationWarning)
```

Comment:
I'm not sure I understand why this had been marked as deprecated (?)

Reply:
I don't know why it had been marked as deprecated either -- assumed it's okay to un-deprecate it when looking at call sites.

Options: 
* leave the `if lamb > 0.5` branch as-is (contributing ~10 lines and some extra state to a loop body in rbfe.py https://github.com/proteneer/timemachine/blob/a5ba0cf76ccf7748134abd7ccc5fd85a22531be9/timemachine/fe/rbfe.py#L145-L155 )
* extract to a free function e.g. `combine_ligand_confs` in rbfe.py (i.e. partially reverting https://github.com/proteneer/timemachine/pull/936/commits/838b9efb347d69da5723797558a0b6b99dfe289f )
* proposed: add optional `lamb` argument to `st.combine_confs`, preserving default behavior

Reply:
I think it's fine as you have it (especially with documentation for lambda). Was mostly just curious if we understood why it had been marked deprecated when it was still being used.